### DISCUSSION TITLE
Wrapped

### DISCUSSION THREAD

**Discussion by CrabGame1** [jpochyla/psst#463] [Q&A]
Hey, i wanted to know if at the end of the year, will the songs played on psst also count for the wrapped?

**Comment by jacksongoode** [comment]
If Spotify has done their homework, they should be since you need to authenticate to retrieve audio. I imagine those requests are logged and are apart of your history. You could request you data from Spotify if you want to confirm this. Try a day playing a lot of specific music and then look for it in their data dump.

However, this is not a planned feature in the scope or concern of this project.
